src/views/groups/GroupEdit/components/EditGroupDiscription.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="b-container">
+    <div class="row">
+      <!-- div class="col-md-8"-->
+      <div class="col-md-12">
+        <h4 class="about-me">{{ $t('group-view:group-info') }}</h4>
+        <p style="font-weight:bold;margin-top:10px">{{ $t('edit-group:group-name') }}</p>
+        <input
+          style="color:#fff"
+          type="text"
+          :placeholder="$t('edit-group:group-name')"
+          :value="personalName"
+          @input="e => $emit('update:personalName', e.target.value)"
+        />
+        <p style="font-weight:bold;margin-top:10px">{{ $t('edit-group:discription') }}</p>
+        <EditField class="editor" :content="aboutMe" @input="text => $emit('update:aboutMe', text)" />
+        <!--
+        <EditPlayerNewSection
+          v-show="addingSection"
+          @set-section="section => $emit('set-section', section)"
+        />
+        <b-button
+          style="margin-top:19px;"
+          @click="addingSection = !addingSection"
+          v-show="!addingSection"
+          variant="secondary"
+        >
+          {{ $t('edit-profile:custom-section-add') }}
+        </b-button>
+        -->
+      </div>
+      <!--div class="col-md-4">
+        <EditPlayerFeaturedAchievement />
+      </div-->
```

Comment:
We can delete all the dead code related to unfinished sections of the user profile